tests/foreman/ui/test_repository.py

```diff
@@ -920,10 +921,18 @@ def test_negative_download_policy_displayed_for_non_yum_repo(self):
 
         @Assert: Dropdown for download policy is not displayed for non-yum repo
         """
-        non_yum_repo_types = [
-            repo_type for repo_type in REPO_TYPE.values()
-            if repo_type != 'yum'
-        ]
+        os_version = get_host_os_version()
```

Comment:
since this should not be a subject of a change across testcases, perhaps we might move this to `setupClass` at least. Ideally, we might have this hardcoded in the settings.